Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs

```diff
@@ -1,7 +1,7 @@
-﻿using Ecommerce.Domain.Entities;
-using Ecommerce.ViewModels.Product;
+﻿using Ecommerce.Application.ViewModels.Product;
+using Ecommerce.Domain.Entities;
 
-namespace Ecommerce.Mappings
+namespace Ecommerce.Application.Mappings
```

Comment:
convert file scoped namespace

Reply:
I don't understand where there is a mistake

Reply:
I think, you got me